views/components/Table/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import {
+  Table as MuiTable,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableRow,
+  Paper,
+} from '@mui/material'
+
+import TableHead from './TableHead'
+
+const Table = (props) => {
+  const {
+    cellRenderer,
+    data,
+    headers,
+    sortDirection,
+    sortProperty,
+    handleRequestSort,
+  } = props
+
+  return (
+    <TableContainer component={Paper}>
+      <MuiTable size="small">
+        <TableHead
+          sortDirection={sortDirection}
+          sortProperty={sortProperty}
+          onRequestSort={handleRequestSort}
+          headCells={headers}
+        />
+        <TableBody>
+          {data.map((rowData) => (
+            <TableRow>
+              {headers.map((header) => (
+                <TableCell key={header.dataProperty}>
```

Comment:
will we always have a dataProperty

Reply:
yeah that's how we'll know what content to render in the cell.